llvm/test/ThinLTO/X86/Inputs/workload2.ll

```diff
@@ -0,0 +1,22 @@
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-pc-linux-gnu"
+
+declare void @m2_variant()
+
+define dso_local void @m2_f1() {
+  call void @interposable_f()
+  call void @noninterposable_f()
+  ret void
+}
+
+@m2_f1_alias = alias void (...), ptr @m2_f1
+
+define linkonce_odr void @interposable_f() {
```

Comment:
Is this meant to have interposable linkage?

Reply:
Well, not *this* one. Its similarly named one in workload1.ll does. The test checks that this copy here will replace the one in workload1.ll.

Reply:
Maybe make it have strong (external) linkage then. I'm not sure in reality that the linker would ever select a linkonce_odr as prevailing over linkonce. Typically it is an interposable (e.g. linkonce or weak) symbol being overridden by a strong symbol.

Reply:
made it external.